herbert_core/utilities/classes/@IX_dataset_2d/ds2.m

```diff
@@ -1,4 +1,4 @@
-function [fig_handle, axes_handle, plot_handle] = ds2(w,varargin)
+function varargout = ds2(w,varargin)
```

Comment:
useful  to say how this differs from ds

Reply:
actually it was there, written at the end of the description

Reply:
but added